### ISSUE TITLE
uniformize help file indents

### ISSUE THREAD

**Issue by Konfekt** [vim/vim#16204]
**Is your feature request about something that is currently impossible or hard to do? Please describe the problem.**

The correct indent of help files with a mix of tabs and spaces may be hard to grasp as a contributing outsider, burdening maintainers with correcting these (as happened to @k-takata du to me in https://github.com/vim/vim/pull/15721/files#diff-49e1f1e8a3b17eb217ce251d8200c88a2c9dede55a4373df3d2176a8403f7f28 )

**Describe the solution you'd like**
Since there's now [uniform formatting of source files](https://github.com/vim/vim/pull/15789#issue-2563506255), how about uniformizing the help file indents (with spaces rather than tabs, say)?

**Describe alternatives you've considered**
Document why tabs and spaces are needed and how to correctly employ them

**Additional context**
See above links

**Comment by dlejay** [comment]
I think the current “style” is forced by `retab!` which has only one objective : compress the file as much as possible.

For the whole of `runtime/doc` the difference (fully expanded vs fully retab!) is **1.20MB**

**Question :** on which systems does saving 1.20MB matter ?

If none, then letʼs stop optimizing for disk size and just expand all tabs.

**Comment by chrisbra** [comment]
I think it matters still. The current runtime doc folder is 12 MB, which is still the reason why it is not distributed with Vim.Tiny (at least on Debian). I'd rather see Debian package the helpfiles even for vim.tiny and for that case, it still matters. CC @jamessan

**Comment by dlejay** [comment]
Perfect, then if it still matters, we just need to add guidelines saying that the official style of the documentation files is `retab!`. What would be the best place for this ?

EDIT : I have found it is probably `develop.txt`

**Comment by jamessan** [comment]
> The current runtime doc folder is 12 MB, which is still the reason why it is not distributed with Vim.Tiny (at least on Debian). I'd rather see Debian package the helpfiles even for vim.tiny and for that case, it still matters.

I did make a change a while back to include uganda.txt, sponsor.txt, and the current versionXY.txt help files, since those are all referenced from the intro screen. After that, it's up to the admin to decide if they want to install vim-runtime to pull in all the runtime content.

**Comment by chrisbra** [comment]
Thanks @jamessan And the reason not to include all is the filesize of the whole vim-runtime package?

**Comment by jamessan** [comment]
Right. From an install media perspective and base install size, there are constraints that are imposed. The vim-common package (which contains the man pages and selected help files) is a 408K .deb and 1.8M installed. The vim-runtime package is a 6.8M .deb and 38.5M installed.

**Comment by dlejay** [comment]
Solved by https://github.com/vim/vim/commit/57d6d004331f1770b5f04ba28ac242c0e0c8a005  ?

**Comment by chrisbra** [comment]
yes, let's close it.
